src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/DeserializationHelpers.cs

```diff
@@ -19,7 +19,15 @@ internal static class DeserializationHelpers
         internal static void ApplyProperty(ODataProperty property, IEdmStructuredTypeReference resourceType, object resource,
             ODataDeserializerProvider deserializerProvider, ODataDeserializerContext readContext)
         {
-            IEdmProperty edmProperty = resourceType.FindProperty(property.Name);
+            IDictionary<string, string> propertyNamesMapping = new Dictionary<string, string>();
+            var sProperties = resourceType.StructuralProperties();
```

Comment:
There's a scenario that we don't support property name case insensitive is that:
Do we consider 'NaMe" and 'Name' two different properties. 'NaMe' could be a dynamic property, how can we distinguish it?

Reply:
@mikepizzo 

---
In reply to: [456543378](https://github.com/OData/WebApi/pull/2233#discussion_r456543378) [](ancestors = 456543378)